lib/spreewald/email_steps.rb

```diff
@@ -65,14 +65,16 @@
   end
 end.overridable
 
+# Please note that this step will only follow HTTP and HTTPS links.
+# Other links (such as mailto: or ftp:// links) are ignored.
```

Comment:
Please add tests for the described cases: Use the step on a Mail with `mailto` and `ftp` links, it should not crash and deliver appropriate results (mailto links are ignored)